source/extensions/quic_listeners/quiche/codec_impl.cc

```diff
@@ -6,14 +6,18 @@
 namespace Envoy {
 namespace Quic {
 
+EnvoyQuicStream* quicStreamToEnvoyStream(quic::QuicStream* stream) {
+  return dynamic_cast<EnvoyQuicStream*>(stream);
```

Comment:
Please add a comment here as was discussed in the previous comment. It's not completely clear to me why we have to have a dynamic cast here and what the inheritance structure is.